session.go

```diff
@@ -196,6 +200,27 @@ GET_ID:
 	return stream, nil
 }
 
+// setOpenTimeout implements a timeout for streams that are opened but not established.
+// If the StreamOpenTimeout is exceeded we assume the server is unable to ACK,
```

Comment:
```suggestion
// If the StreamOpenTimeout is exceeded we assume the peer is unable to ACK,
```
Yamux can open streams in either direction not just client to server in the TCP sense.